src/components/Reports/ProjectReport/__tests__/ProjectReport.test.js

```diff
@@ -225,39 +225,36 @@ describe('ProjectReport component', () => {
 
 describe('ProjectReport WBS link visibility', () => {
   it(`should display WBS links when the user has required permissions`, async () => {
-    const mockPermissions = ['resolveTask', 'acb'];
-
+    const mockPermissions = ['viewWBS'];
     const hasPermission = mockPermissions.some(permission =>
       viewWBSpermissionsRequired.includes(permission),
     );
 
     const canViewWBS = hasPermission;
 
-    axios.get.mockResolvedValue({ status: 200, data: {} });
+    const mockWBS = { _id: 'wbs123', wbsName: 'wbs name1' };
+  const projectId = '123';
+
+    axios.get.mockResolvedValue({ status: 200, data: { mockWBS }});
 
     render(
       <Provider store={store}>
         <ProjectReport />
       </Provider>,
     );
 
-    const mockWBS = { _id: 'wbs123', wbsName: 'wbs name1' };
-    const projectId = '123';
-
     if (canViewWBS) {
-      screen.findByRole('link', { name: mockWBS.wbsName }).then(linkElement => {
+      const linkElement = await screen.findByRole('link', { name: mockWBS.wbsName });
         expect(linkElement).toBeInTheDocument();
         expect(linkElement).toHaveAttribute(
           'href',
           expect.stringContaining(`/wbs/tasks/${mockWBS._id}/${projectId}/${mockWBS.wbsName}`),
         );
-      });
     } else {
-      screen.findByText(mockWBS.wbsName).then(divElement => {
```

Comment:
Hey Carlos,
I was not able to grasp the change made at line 256.
Why was this test removed?

Reply:
Hi Parth.
it was fixed, please check it _again._
thanks.